be/src/column/column_visitor_mutable.h

```diff
@@ -40,6 +40,7 @@ class ColumnVisitorMutable {
     virtual Status visit(vectorized::HyperLogLogColumn* column);
     virtual Status visit(vectorized::BitmapColumn* column);
     virtual Status visit(vectorized::PercentileColumn* column);
+    virtual Status visit(vectorized::JsonColumn* column);
```

Comment:
what the difference betweens JsonColumn and ObjectColumn<JsonValue>?

Reply:
Currently, JsonColumn just inherits ObjectColumn, and overwrites some `put_mysql_buffer`, hash methods.

Even more important, JsonColumn needs to implement its own serialization and access algorithms, more closely to columnar storage, not only ObjectColumn.